packages/system/cert-manager/Makefile

```diff
@@ -1,17 +1,10 @@
 NAME=cert-manager
-NAMESPACE=cozy-cert-manager
+NAMESPACE=cozy-$(NAME)
 
-show:
-	helm template --dry-run=server -n $(NAMESPACE) $(NAME) .
-
-apply:
-	helm upgrade -i -n $(NAMESPACE) $(NAME) .
-
-diff:
-	helm diff upgrade --allow-unreleased --normalize-manifests -n $(NAMESPACE) $(NAME) .
+include ../../hack/package-system.mk
 
 update:
 	rm -rf charts
 	helm repo add jetstack https://charts.jetstack.io
 	helm repo update jetstack
-	helm pull jetstack/cert-manager --untar --untardir charts
+	helm pull jetstack/$(NAME) --untar --untardir charts
```

Comment:
I would rather keep update target without using variables, because commands here targets to concrete repositories, they are node depended on release name or namespace.